tests/Integration/PrestaShopBundle/Routing/Converter/LegacyUrlConverterTest.php

```diff
@@ -173,6 +173,10 @@ public static function getMigratedControllers()
             'admin_metas_set_up_urls_save' => ['/configure/shop/seo-urls/set-up-urls', 'AdminMeta', 'submitOptionsmeta'],
             'admin_metas_generate_robots_text_file' => ['/configure/shop/seo-urls/generate/robots', 'AdminMeta', 'submitRobots'],
 
+            'admin_permissions_index' => ['/configure/advanced/permissions/', 'AdminAccess'],
+            'admin_permissions_update_tab_permissions' => ['/configure/advanced/permissions/update/permissions/tab', 'AdminAccess', 'updateAccess'],
+            'admin_permissions_update_module_permissions' => ['/configure/advanced/permissions/update/permissions/module', 'AdminAccess', 'updateModuleAccess'],
+
             //'admin_module_configure_action' => ['/improve/modules/manage/action/configure/ps_linklist', 'AdminModules', 'configure', ['module_name' => 'ps_linklist']],
```

Comment:
why keep these comments ?

Reply:
It's not related to the pull request :thinking: